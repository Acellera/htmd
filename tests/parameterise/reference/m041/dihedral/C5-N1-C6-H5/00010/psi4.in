set {
	reference rhf
	basis 6-31G*
}


set_num_threads( 1 )
memory 2 gb

molecule MOL {
0 1

H	 -2.271982	 -1.317245	 1.447150
C	 -1.570325	 -0.788909	 0.832377
C	 -1.961029	 0.349207	 0.051630
C	 -0.289512	 -1.174794	 0.768923
H	 -2.982895	 0.683806	 0.081274
C	 -1.065454	 0.993922	 -0.709784
H	 0.082878	 -2.016036	 1.322133
N	 0.640212	 -0.532897	 -0.000111
H	 -1.321048	 1.847858	 -1.306291
C	 0.328237	 0.573841	 -0.778623
C	 2.010177	 -1.032049	 0.000042
O	 1.163570	 1.126303	 -1.450344
H	 2.187698	 -1.635381	 -0.882279
H	 2.677579	 -0.189247	 -0.001871
H	 2.188882	 -1.632203	 0.884288

	symmetry c1
}
ee = energy('scf')
oeprop('DIPOLE', 'QUADRUPOLE', 'MULLIKEN_CHARGES')
f=open( 'psi4out.xyz', 'w' )
f.write( "15  " )
f.write( str(ee) + "\n" )
f.write( MOL.save_string_xyz() )
f.close()
