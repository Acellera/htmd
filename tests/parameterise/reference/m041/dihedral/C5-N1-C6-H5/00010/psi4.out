    -----------------------------------------------------------------------
          Psi4: An Open-Source Ab Initio Electronic Structure Package
                              Psi4 (no tag) Driver

                          Git: Rev {detached?}  

    J. M. Turney, A. C. Simmonett, R. M. Parrish, E. G. Hohenstein,
    F. A. Evangelista, J. T. Fermann, B. J. Mintz, L. A. Burns, J. J. Wilke,
    M. L. Abrams, N. J. Russ, M. L. Leininger, C. L. Janssen, E. T. Seidl,
    W. D. Allen, H. F. Schaefer, R. A. King, E. F. Valeev, C. D. Sherrill,
    and T. D. Crawford, WIREs Comput. Mol. Sci. 2, 556-565 (2012)
    (doi: 10.1002/wcms.93)

                         Additional Contributions by
    A. E. DePrince, M. Saitow, U. Bozkaya, A. Yu. Sokolov
    -----------------------------------------------------------------------


    Psi4 started on: Tue Jul  5 16:30:00 2016

    Process ID:  28510
    PSI4DATADIR: /export131/home/mjharvey/miniconda3/share
    Memory level set to 256.000 MB

  ==> Input File <==

--------------------------------------------------------------------------
set {
	reference rhf
	basis 6-31G*
}


set_num_threads( 1 )
memory 2 gb

molecule MOL {
0 1

H	 -2.271982	 -1.317245	 1.447150
C	 -1.570325	 -0.788909	 0.832377
C	 -1.961029	 0.349207	 0.051630
C	 -0.289512	 -1.174794	 0.768923
H	 -2.982895	 0.683806	 0.081274
C	 -1.065454	 0.993922	 -0.709784
H	 0.082878	 -2.016036	 1.322133
N	 0.640212	 -0.532897	 -0.000111
H	 -1.321048	 1.847858	 -1.306291
C	 0.328237	 0.573841	 -0.778623
C	 2.010177	 -1.032049	 0.000042
O	 1.163570	 1.126303	 -1.450344
H	 2.187698	 -1.635381	 -0.882279
H	 2.677579	 -0.189247	 -0.001871
H	 2.188882	 -1.632203	 0.884288

	symmetry c1
}
ee = energy('scf')
oeprop('DIPOLE', 'QUADRUPOLE', 'MULLIKEN_CHARGES')
f=open( 'psi4out.xyz', 'w' )
f.write( "15  " )
f.write( str(ee) + "\n" )
f.write( MOL.save_string_xyz() )
f.close()
--------------------------------------------------------------------------

  Memory set to   2.000 GiB by Python script.

*** tstart() called on cx1-3-2-3.cx1.hpc.ic.ac.uk
*** at Tue Jul  5 16:30:00 2016


         ---------------------------------------------------------
                                   SCF
            by Justin Turney, Rob Parrish, and Andy Simmonett
                              RHF Reference
                        1 Threads,   2000 MiB Core
         ---------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c1
    Full point group: C1

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z               Mass       
    ------------   -----------------  -----------------  -----------------  -----------------
           H         -2.249669585568    -1.255951456733     1.627506857407     1.007825032070
           C         -1.548012585568    -0.727615456733     1.012733857407    12.000000000000
           C         -1.938716585568     0.410500543267     0.231986857407    12.000000000000
           C         -0.267199585568    -1.113500456733     0.949279857407    12.000000000000
           H         -2.960582585568     0.745099543267     0.261630857407     1.007825032070
           C         -1.043141585568     1.055215543267    -0.529427142593    12.000000000000
           H          0.105190414432    -1.954742456733     1.502489857407     1.007825032070
           N          0.662524414432    -0.471603456733     0.180245857407    14.003074004780
           H         -1.298735585568     1.909151543267    -1.125934142593     1.007825032070
           C          0.350549414432     0.635134543267    -0.598266142593    12.000000000000
           C          2.032489414432    -0.970755456733     0.180398857407    12.000000000000
           O          1.185882414432     1.187596543267    -1.269987142593    15.994914619560
           H          2.210010414432    -1.574087456733    -0.701922142593     1.007825032070
           H          2.699891414432    -0.127953456733     0.178485857407     1.007825032070
           H          2.211194414432    -1.570909456733     1.064644857407     1.007825032070

  Running in c1 symmetry.

  Rotational constants: A =      0.11492  B =      0.07866  C =      0.04710 [cm^-1]
  Rotational constants: A =   3445.17233  B =   2358.02694  C =   1412.15428 [MHz]
  Nuclear repulsion =  357.507515274579248

  Charge       = 0
  Multiplicity = 1
  Electrons    = 58
  Nalpha       = 29
  Nbeta        = 29

  ==> Algorithm <==

  SCF Algorithm Type is DF.
  DIIS enabled.
  MOM disabled.
  Fractional occupation disabled.
  Guess Type is CORE.
  Energy threshold   = 1.00e-06
  Density threshold  = 1.00e-06
  Integral threshold = 0.00e+00

  ==> Primary Basis <==

  Basis Set: 6-31G*
    Number of shells: 62
    Number of basis function: 134
    Number of Cartesian functions: 134
    Spherical Harmonics?: false
    Max angular momentum: 2

  ==> Pre-Iterations <==

   -------------------------------------------------------
    Irrep   Nso     Nmo     Nalpha   Nbeta   Ndocc  Nsocc
   -------------------------------------------------------
     A        134     134       0       0       0       0
   -------------------------------------------------------
    Total     134     134      29      29      29       0
   -------------------------------------------------------

 OEINTS: Overlap, kinetic, potential, dipole, and quadrupole integrals
         stored in file 35.

  ==> Integral Setup <==

  ==> DFJK: Density-Fitted J/K Matrices <==

    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    OpenMP threads:              1
    Integrals threads:           1
    Memory (MB):              1430
    Algorithm:                Core
    Integral Cache:           NONE
    Schwarz Cutoff:          1E-12
    Fitting Condition:       1E-12

   => Auxiliary Basis Set <=

  Basis Set: 
    Number of shells: 255
    Number of basis function: 823
    Number of Cartesian functions: 823
    Spherical Harmonics?: false
    Max angular momentum: 3

  Minimum eigenvalue in the overlap matrix is 7.0483260054E-04.
  Using Symmetric Orthogonalization.
  SCF Guess: Core (One-Electron) Hamiltonian.

  ==> Iterations <==

                           Total Energy        Delta E     RMS |[F,P]|

   @DF-RHF iter   1:  -284.70715753428789   -2.84707e+02   6.98777e-02 
   @DF-RHF iter   2:  -277.33584561205174    7.37131e+00   5.28189e-02 DIIS
   @DF-RHF iter   3:  -320.95360051775782   -4.36178e+01   5.10976e-02 DIIS
   @DF-RHF iter   4:  -355.99252101166798   -3.50389e+01   1.84009e-02 DIIS
   @DF-RHF iter   5:  -358.50893035139876   -2.51641e+00   1.38393e-02 DIIS
   @DF-RHF iter   6:  -360.52164247848140   -2.01271e+00   2.10033e-03 DIIS
   @DF-RHF iter   7:  -360.58332300575336   -6.16805e-02   8.04692e-04 DIIS
   @DF-RHF iter   8:  -360.59200447894790   -8.68147e-03   3.50855e-04 DIIS
   @DF-RHF iter   9:  -360.59476320250536   -2.75872e-03   1.99137e-04 DIIS
   @DF-RHF iter  10:  -360.59579814745661   -1.03494e-03   6.52909e-05 DIIS
   @DF-RHF iter  11:  -360.59591638649437   -1.18239e-04   1.94462e-05 DIIS
   @DF-RHF iter  12:  -360.59592727460864   -1.08881e-05   1.13345e-05 DIIS
   @DF-RHF iter  13:  -360.59593029200067   -3.01739e-06   3.79616e-06 DIIS
   @DF-RHF iter  14:  -360.59593073864318   -4.46643e-07   9.83021e-07 DIIS

  ==> Post-Iterations <==

    Orbital Energies (a.u.)
    -----------------------

    Doubly Occupied:                                                      

       1A    -20.509548     2A    -15.622459     3A    -11.343063  
       4A    -11.296857     5A    -11.267712     6A    -11.259894  
       7A    -11.232740     8A    -11.228557     9A     -1.384118  
      10A     -1.281827    11A     -1.114595    12A     -1.025636  
      13A     -0.971064    14A     -0.872648    15A     -0.835561  
      16A     -0.715023    17A     -0.696872    18A     -0.662123  
      19A     -0.649369    20A     -0.621644    21A     -0.606863  
      22A     -0.574440    23A     -0.550954    24A     -0.546275  
      25A     -0.527874    26A     -0.477323    27A     -0.414009  
      28A     -0.405795    29A     -0.304605  

    Virtual:                                                              

      30A      0.111762    31A      0.180644    32A      0.226614  
      33A      0.252489    34A      0.280393    35A      0.308788  
      36A      0.312099    37A      0.325325    38A      0.331671  
      39A      0.343482    40A      0.367290    41A      0.433635  
      42A      0.468300    43A      0.483487    44A      0.508263  
      45A      0.535231    46A      0.635697    47A      0.690520  
      48A      0.695044    49A      0.731912    50A      0.750764  
      51A      0.755579    52A      0.792412    53A      0.802737  
      54A      0.809457    55A      0.821481    56A      0.844129  
      57A      0.859468    58A      0.894076    59A      0.938888  
      60A      0.956761    61A      0.977461    62A      1.000984  
      63A      1.030596    64A      1.067408    65A      1.100363  
      66A      1.112202    67A      1.127070    68A      1.143763  
      69A      1.170488    70A      1.173304    71A      1.194941  
      72A      1.208210    73A      1.225265    74A      1.251012  
      75A      1.291978    76A      1.311749    77A      1.359321  
      78A      1.405652    79A      1.452489    80A      1.470783  
      81A      1.483094    82A      1.519348    83A      1.609975  
      84A      1.652640    85A      1.681460    86A      1.704599  
      87A      1.718777    88A      1.744106    89A      1.766889  
      90A      1.769270    91A      1.945640    92A      2.005452  
      93A      2.070175    94A      2.113831    95A      2.174503  
      96A      2.186754    97A      2.210114    98A      2.245083  
      99A      2.253262   100A      2.287671   101A      2.325589  
     102A      2.352158   103A      2.364310   104A      2.439409  
     105A      2.444872   106A      2.497687   107A      2.504078  
     108A      2.554423   109A      2.603563   110A      2.646716  
     111A      2.677605   112A      2.700632   113A      2.709261  
     114A      2.741966   115A      2.922174   116A      2.949445  
     117A      3.019402   118A      3.034976   119A      3.048554  
     120A      3.098001   121A      3.166351   122A      3.201562  
     123A      3.257503   124A      3.405713   125A      3.602359  
     126A      3.820436   127A      4.452480   128A      4.501354  
     129A      4.555774   130A      4.646804   131A      4.693583  
     132A      4.763145   133A      4.925934   134A      5.153700  

    Final Occupation by Irrep:
              A 
    DOCC [    29 ]

  Energy converged.

  @DF-RHF Final Energy:  -360.59593073864318

   => Energetics <=

    Nuclear Repulsion Energy =            357.5075152745792479
    One-Electron Energy =               -1198.2081228196500433
    Two-Electron Energy =                 480.1046768064275057
    DFT Exchange-Correlation Energy =       0.0000000000000000
    Empirical Dispersion Energy =           0.0000000000000000
    PCM Polarization Energy =               0.0000000000000000
    EFP Energy =                            0.0000000000000000
    Total Energy =                       -360.5959307386433466

    Alert: EFP and PCM quantities not currently incorporated into SCF psivars.

Properties will be evaluated at   0.000000,   0.000000,   0.000000 Bohr
  ==> Properties <==


Properties computed using the SCF density matrix

  Nuclear Dipole Moment: (a.u.)
     X:     0.6762      Y:    -3.5829      Z:     2.6248

  Electronic Dipole Moment: (a.u.)
     X:    -1.2749      Y:     2.3977      Z:    -1.5016

  Dipole Moment: (a.u.)
     X:    -0.5986      Y:    -1.1852      Z:     1.1232     Total:     1.7391

  Dipole Moment: (Debye)
     X:    -1.5216      Y:    -3.0124      Z:     2.8548     Total:     4.4204


  Saving occupied orbitals to File 180.

*** tstop() called on cx1-3-2-3.cx1.hpc.ic.ac.uk at Tue Jul  5 16:30:11 2016
Module time:
	user time   =      10.41 seconds =       0.17 minutes
	system time =       0.06 seconds =       0.00 minutes
	total time  =         11 seconds =       0.18 minutes
Total time:
	user time   =      10.41 seconds =       0.17 minutes
	system time =       0.06 seconds =       0.00 minutes
	total time  =         11 seconds =       0.18 minutes


Properties will be evaluated at   0.000000,   0.000000,   0.000000 Bohr

Properties computed using the  density matrix

  Nuclear Dipole Moment: (a.u.)
     X:     0.6762      Y:    -3.5829      Z:     2.6248

  Electronic Dipole Moment: (a.u.)
     X:    -1.2749      Y:     2.3977      Z:    -1.5016

  Dipole Moment: (a.u.)
     X:    -0.5986      Y:    -1.1852      Z:     1.1232     Total:     1.7391

  Dipole Moment: (Debye)
     X:    -1.5216      Y:    -3.0124      Z:     2.8548     Total:     4.4204

  Quadrupole Moment: (Debye Ang)
    XX:   -42.4181     YY:   -46.5232     ZZ:   -49.7543
    XY:    -6.1944     XZ:     3.2036     YZ:    -0.1308

  Traceless Quadrupole Moment: (Debye Ang)
    XX:     3.8138     YY:    -0.2914     ZZ:    -3.5224
    XY:    -6.1944     XZ:     3.2036     YZ:    -0.1308

  Mulliken Charges: (a.u.)
   Center  Symbol    Alpha    Beta     Spin     Total
       1     H     0.39311  0.39311  0.00000  0.21379
       2     C     3.16163  3.16163  0.00000 -0.32326
       3     C     3.05836  3.05836  0.00000 -0.11671
       4     C     2.92257  2.92257  0.00000  0.15486
       5     H     0.39096  0.39096  0.00000  0.21807
       6     C     3.17020  3.17020  0.00000 -0.34041
       7     H     0.38791  0.38791  0.00000  0.22419
       8     N     3.86487  3.86487  0.00000 -0.72975
       9     H     0.38649  0.38649  0.00000  0.22701
      10     C     2.59484  2.59484  0.00000  0.81031
      11     C     3.14313  3.14313  0.00000 -0.28626
      12     O     4.32042  4.32042  0.00000 -0.64083
      13     H     0.40344  0.40344  0.00000  0.19313
      14     H     0.38818  0.38818  0.00000  0.22364
      15     H     0.41388  0.41388  0.00000  0.17224

   Total alpha = 29.00000, Total beta = 29.00000, Total charge =  0.00000


*** PSI4 exiting successfully. Buy a developer a beer!
